base/system/chkdsk/lang/ro-RO.rc

```diff
@@ -1,17 +1,18 @@
 /*
- * PROJECT:     Check Disk utility
- * LICENSE:     See COPYING in the top level directory
+ * PROJECT:     ReactOS Disk Checking Utility
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
  * PURPOSE:     Romanian resource file
- * COPYRIGHT:   Copyright 2018 George Bișoc (george.bisoc@reactos.org)
+ * TRANSLATORS: Copyright 2018 George Bișoc <george.bisoc@reactos.org>
+ *              Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
     IDS_USAGE "\n\
-Uz: %s [drive:] [-F] [-V] [-R] [-C]\n\n\
-[drive:]    Specifică unitatea de stocare pentru control.\n\
+Uz: %s [disc:] [-F] [-V] [-R] [-C]\n\n\
+[disc:]     Specifică unitatea de stocare pentru control.\n\
```

Comment:
I would prefer the adoption of `dispozitiv` since the wide usage of other storage mediums like SSDs which are not HDDs on themselves. Of course this would mean updating the whole translation file that uses the word `disc`.

Reply:
But at the same time, the "D" in e.g. SSD means "Disc"

Reply:
@GeoB99 I also want to add that "dispozitiv" means "device" in English (also see MSDN). We reffer to C: drive and not to C: device. **A device CAN have more volumes** so it's NOT mandatory to have just one. As @HBelusca mentioned I want to tell you that MSDN is my main source of inspiration besides translations for ReactOS in other languages. I don't know when this rule appeared and where did it come from (to talk about devices instead of drives or volumes for Romanian language) but I'm 100% sure it wasn't invented neither by me or by MSDN.